providers/discovery/external/external.go

```diff
@@ -40,12 +40,13 @@ type Response struct {
 
 // Request is the request sent to the external script on its STDIN
 type Request struct {
-	Protocol   string            `json:"protocol"`
-	Collective string            `json:"collective"`
-	Filter     *protocol.Filter  `json:"filter"`
-	Options    map[string]string `json:"options"`
-	Schema     string            `json:"$schema"`
-	Timeout    float64           `json:"timeout"`
+	Protocol    string            `json:"protocol"`
+	Collective  string            `json:"collective"`
+	Filter      *protocol.Filter  `json:"filter"`
+	Federations []string          `json:"federations"`
```

Comment:
This will require updating the schema, unsure if this is what you meant or if it should be something in filter

Reply:
no this is good, exactly what I had in mind.  We already pass Collective there so it seemed a related good fit.